ui/common/src/main/res/values/strings.xml

```diff
@@ -325,7 +325,9 @@
   <string name="transaction_rejected_body">This purchase has been rejected.\nFor more information contact support.</string>
   <string name="transaction_reverted_body">This purchase has been reverted.\nFor more information contact support.</string>
   <string name="transaction_status_rejected">Rejected</string>
-  <string name="transaction_details_header">Transaction detail</string>
+  <string name="transaction_details_header">Transaction details</string>
+  <string name="transaction_details_txid">TxID</string>
+  <string name="transaction_details_sku">SKU</string>
  
```

Comment:
should we add strings? is not better asking for carlos?

Reply:
They were added by Carlos for dev branch, at the time I added here because of the conflicts between dev and wallet_v3